peppy/sample.py

```diff
@@ -320,17 +301,37 @@ def project(self):
         """
         return self[PRJ_REF]
 
-    def __setattr__(self, key, value):
-        self._try_touch_samples()
-        super(Sample, self).__setattr__(key, value)
-
     def __delattr__(self, item):
         self._try_touch_samples()
         super(Sample, self).__delattr__(item)
 
-    def __setitem__(self, key, value):
+    def __setitem__(self, item, value):
```

Comment:
why did you rename this from `key` to `item` ? I think `key` is more clear, I'd call 'item' the key:value pair.

Reply:
I think, I was changing code few time and didn't mentioned this. You are right key is more clear